### PR TITLE
Polidea update - Blog

### PR DESCRIPTION
**Pull request by Jakub-Sadowski**
PR contains list of blog's posts page and single blog's post page.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[X] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by Jakub-Sadowski** [comment]
Review: @davidhauck

**Comment by craigbox** [comment]
Overview page:

- clicking the years puts a border around the buttons
- it's counterintuitive that all unselected is "everything" and you can select multiple years
- each post has a subtitle, which should be the text on the main page

Single blog post:

- big gap between heading and subheading
- not enough space between "Date | By X"; maybe the bar should be darker or the whole thing should be centered
- larger typeface/line spacing?  See the difference between first and second paragraphs in the XD. I would make all body text the former.
- too much space under each heading
- link to previous/next blog post not at bottom any more

**Comment by Jakub-Sadowski** [comment]
> Overview page:
> 
> * clicking the years puts a border around the buttons

Deleting this border is not recommended as it regresses the accessibility of the site. By default, that blue outline is placed on focusable elements. This is so that users with accessibility issues are able to focus that button by tabbing to it. Some users do not have the motor skills to use a mouse and must use only the keyboard (or some other input device) for computer interaction. When removed, there is no longer a visual indicator on what element is focused.

**Comment by Jakub-Sadowski** [comment]
> Overview page:
> 
> * each post has a subtitle, which should be the text on the main page

We need to discuss this one, because design looks like below and subtitles aren't in every posts and have different lengths so it can be troublesome to place it.
image

**Comment by Jakub-Sadowski** [comment]
> Overview page:
> 
> * it's counterintuitive that all unselected is "everything" and you can select multiple years
> 
So if from the start the button with the recent year will be selected, then it will be resolved? If not could you expand this topic?

**Comment by Jakub-Sadowski** [comment]
> Single blog post:
> 
> * too much space under each heading
> * link to previous/next blog post not at bottom any more

Spacings around headings in content are globally set in whole webpage according to desings, are you sure about changing it ?

Links weren't on designs anymore, but I added them now.

File: layouts/blog/list.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{{ define "main" }}
+<div class="blog-content">
+    <h1>Blog</h1>
+    <p class="blog-description">Istio addresses the challenges developers and operators face as monolithic applications transition towards a distributed microservice architecture. </p>
```
[the remark this answers is not in the document]
Comment (Jakub-Sadowski) [reply layouts/blog/list.html:4]: Sure, I added it

File: content/en/news/index.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+To add new blog post, go to appropriete year folder in content/en/blog/ and create new folder with index.md file similar to existing ones in neighbouring folders. New post will be displayed imidietly after.
```
[the remark this answers is not in the document]
Comment (Jakub-Sadowski) [reply content/en/news/index.md:1]: I'm sorry it was part of documentation and it shouldn't be here, I deleted this file

**Comment by craigbox** [comment]
agreed to change body text size to 18 with TODO on where to change back to 16 if we choose, and then will merge.

**Comment by craigbox** [comment]
Should the year buttons not be stacked (or smaller) or something, on mobile?  My point was, just scrolling them doesn't feel responsive.

**Comment by Jakub-Sadowski** [comment]
> Should the year buttons not be stacked (or smaller) or something, on mobile? My point was, just scrolling them doesn't feel responsive.

This was made according to the design, from my experience behaviour of this buttons is pretty common

**Review by craigbox — APPROVED** [review]

**Pull request merged**